frontend/src/components/input.tsx

```diff
@@ -20,6 +21,11 @@ export interface InputProps {
 
 export const Input = ({ sendMessage, waiting, suggestions }: InputProps) => {
   const [userInput, setUserInput] = useState<string>('');
+  const [uploadedFile, setUploadedFile] = useState<File | null>(null);
+  const [isUploading, setIsUploading] = useState<boolean>(false);
+  const [isUploaded, setIsUploaded] = useState<boolean>(false);
+  // TODO: Add fileId to the state
+  // const [fileId, setFileId] = useState<string>('');
```

Comment:
Is this for future use?

Reply:
Yes, if we needed fileId, but there are other ways and depends upon the design to get that report, so I have removed it.